app/upload.go

```diff
@@ -154,8 +154,8 @@ func (a *App) CreateUploadSession(c request.CTX, us *model.UploadSession) (*mode
 	return us, nil
 }
 
-func (a *App) GetUploadSession(uploadId string) (*model.UploadSession, *model.AppError) {
-	us, err := a.Srv().Store().UploadSession().Get(uploadId)
+func (a *App) GetUploadSession(c *request.Context, uploadId string) (*model.UploadSession, *model.AppError) {
```

Comment:
We usually accept the `request.CTX` interface rather than the actual struct. That keeps things encapsulated.